atomix/cluster/src/test/java/io/atomix/raft/RaftTest.java

```diff
@@ -108,21 +75,18 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 import org.mockito.Mockito;
-import org.slf4j.LoggerFactory;
 
 /** Raft test. */
 public class RaftTest extends ConcurrentTestCase {
```

Comment:
Is it difficult to use `RaftRule` for this also?

Reply:
Probably not. I used most of the setup code from there, but I wanted to minimize the changes :sweat_smile: But yes I can migrated them as well. I had hoped that we can replace most of them step by step.

Reply:
I put it on my agenda of my next issues where I also work on the tests.